src/use_case/save_places/SavePlacesDataAccessObject.java

```diff
@@ -0,0 +1,5 @@
+package use_case.save_places;
+
+public interface SavePlacesDataAccessObject {
```

Comment:
[Naming Issue]
By convention, this file should be named with "SavePlacesDataAccessInterface". DAO should only exists in "data_access" folder.